src/integrationTest/groovy/wooga/gradle/appcenter/AppCenterPluginIntegrationSpec.groovy

```diff
@@ -74,59 +74,60 @@ class AppCenterPluginIntegrationSpec extends IntegrationSpec {
+        "apiToken"              | "'zzz'"                      | "zzz"                                    | "zzz"             | PropertyLocation.script
+        "apiToken"              | null                         | _                                        | "null"            | PropertyLocation.none
+
+        "owner"                 | "xxx"                        | _                                        | "xxx"             | PropertyLocation.env
+        "owner"                 | "yyy"                        | _                                        | "yyy"             | PropertyLocation.property
+        "owner"                 | "'zzz'"                      | "zzz"                                    | "zzz"             | PropertyLocation.script
+        "owner"                 | null                         | _                                        | "null"            | PropertyLocation.none
+
+        "applicationIdentifier" | "xxx"                        | _                                        | "xxx"             | PropertyLocation.env
+        "applicationIdentifier" | "yyy"                        | _                                        | "yyy"             | PropertyLocation.property
+        "applicationIdentifier" | "'zzz'"                      | "zzz"                                    | "zzz"             | PropertyLocation.script
+        "applicationIdentifier" | null                         | _                                        | "null"            | PropertyLocation.none
+
+        "defaultDestinations"   | "group1"                     | [["name": "group1"]]                     | "group1"          | PropertyLocation.env
+        "defaultDestinations"   | "group1,group2"              | [["name": "group1"], ["name": "group2"]] | "group1,group2"   | PropertyLocation.env
+        "defaultDestinations"   | "group1, group2"             | [["name": "group1"], ["name": "group2"]] | "group1, group2"  | PropertyLocation.env
+        "defaultDestinations"   | "group2"                     | [["name": "group2"]]                     | "group2"          | PropertyLocation.property
+        "defaultDestinations"   | "group2,group3"              | [["name": "group2"], ["name": "group3"]] | "group2,group3"   | PropertyLocation.property
+        "defaultDestinations"   | "group2, group3"             | [["name": "group2"], ["name": "group3"]] | "group2, group3"  | PropertyLocation.property
+        "defaultDestinations"   | "['group3']"                 | [["name": "group3"]]                     | "[group3]"        | PropertyLocation.script
+        "defaultDestinations"   | "['group3','group4']"        | [["name": "group3"], ["name": "group4"]] | "[group3,group4]" | PropertyLocation.script
+        "defaultDestinations"   | "'group3,group4'.split(',')" | [["name": "group3"], ["name": "group4"]] | "String..."       | PropertyLocation.script
```

Comment:
this is the place where I added a new test case. Sorry for the reformat.

Reply:
Haha, ok, I was surprised why the change set is so big.